### ISSUE TITLE
Unrelated .yamllint rules affects ansible-lint

### ISSUE THREAD

**Issue by konstruktoid** [ansible/ansible-lint#3101]
##### Summary

```sh
$ ansible-lint --version
ansible-lint 6.13.1 using ansible 2.14.2
$ cat .yamllint.yml 
---
extends: default
ignore: ".tox*\n.git*"
rules:
  line-length:
    max: 120
    level: warning
$ ansible-lint .
WARNING  Replaced outdated tag 'var-spacing' with 'jinja[spacing]', replace it to avoid future regressions
WARNING  Listing 6 violation(s) that are fatal
yaml[line-length]: Line too long (290 > 120 characters) (warning)
defaults/main/sshd.yml:18

yaml[line-length]: Line too long (143 > 120 characters) (warning)
defaults/main/sshd.yml:21

yaml[line-length]: Line too long (129 > 120 characters) (warning)
tasks/packagemgmt.yml:168

yaml[line-length]: Line too long (162 > 120 characters) (warning)
tasks/packagemgmt.yml:198

yaml[line-length]: Line too long (121 > 120 characters) (warning)
tasks/post.yml:22

yaml[line-length]: Line too long (143 > 120 characters) (warning)
tasks/sshconfig.yml:14

Read documentation for instructions on how to ignore specific rule violations.

                   Rule Violation Summary                   
 count tag               profile rule associated tags       
     6 yaml[line-length] basic   formatting, yaml (warning) 

Passed with min profile: 0 failure(s), 6 warning(s) on 70 files.
$ git rm .yamllint.yml 
rm '.yamllint.yml'
$ ansible-lint .
WARNING  Replaced outdated tag 'var-spacing' with 'jinja[spacing]', replace it to avoid future regressions
WARNING  Listing 113 violation(s) that are fatal
yaml[line-length]: Line too long (290 > 160 characters) (warning)
defaults/main/sshd.yml:18

yaml[octal-values]: Forbidden implicit octal value "077"
defaults/main/umask.yml:2

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/adduser.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/adduser.yml:21

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/aide.yml:39

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/aide.yml:52

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/apparmor.yml:9

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/apparmor.yml:26

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/apport.yml:8

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:9

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:37

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:50

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:62

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:75

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:88

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:101

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:115

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:127

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:139

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:152

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/auditd.yml:165

yaml[octal-values]: Forbidden implicit octal value "0600"
tasks/auditd.yml:176

yaml[octal-values]: Forbidden implicit octal value "0750"
tasks/compilers.yml:20

yaml[octal-values]: Forbidden implicit octal value "0600"
tasks/cron.yml:34

yaml[octal-values]: Forbidden implicit octal value "0700"
tasks/cron.yml:61

yaml[octal-values]: Forbidden implicit octal value "0600"
tasks/cron.yml:82

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/disablefs.yml:7

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/disablemod.yml:7

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/disablemod.yml:47

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/disablenet.yml:7

yaml[octal-values]: Forbidden implicit octal value "0755"
tasks/facts.yml:8

yaml[octal-values]: Forbidden implicit octal value "0755"
tasks/facts.yml:19

yaml[octal-values]: Forbidden implicit octal value "0755"
tasks/facts.yml:31

yaml[octal-values]: Forbidden implicit octal value "0755"
tasks/facts.yml:44

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/hosts.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/hosts.yml:20

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/ipv6.yml:9

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/issue.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/issue.yml:23

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/journalconf.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/journalconf.yml:26

yaml[octal-values]: Forbidden implicit octal value "0750"
tasks/journalconf.yml:37

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/journalconf.yml:69

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/journalconf.yml:83

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/limits.yml:9

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/limits.yml:25

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/logindconf.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/logindefs.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/motdnews.yml:16

yaml[octal-values]: Forbidden implicit octal value "0444"
tasks/motdnews.yml:40

yaml[octal-values]: Forbidden implicit octal value "0755"
tasks/motdnews.yml:51

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/mount.yml:47

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/packagemgmt.yml:6

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/packagemgmt.yml:84

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/packagemgmt.yml:101

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/packagemgmt.yml:116

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/packagemgmt.yml:132

yaml[line-length]: Line too long (162 > 160 characters) (warning)
tasks/packagemgmt.yml:198

yaml[octal-values]: Forbidden implicit octal value "0755"
tasks/packages.yml:26

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/packages.yml:52

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/packages.yml:206

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:28

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:51

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:66

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:81

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:94

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:110

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:122

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:134

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:146

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:160

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:201

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:264

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/password.yml:278

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/path.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/path.yml:22

yaml[octal-values]: Forbidden implicit octal value "0755"
tasks/post.yml:7

yaml[octal-values]: Forbidden implicit octal value "0400"
tasks/post.yml:83

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/prelink.yml:21

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/resolvedconf.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/resolvedconf.yml:23

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/rkhunter.yml:8

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/rkhunter.yml:25

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/rkhunter.yml:42

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/rootaccess.yml:23

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/rootaccess.yml:34

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/rootaccess.yml:55

yaml[octal-values]: Forbidden implicit octal value "0600"
tasks/sshconfig.yml:66

yaml[octal-values]: Forbidden implicit octal value "0600"
tasks/sshconfig.yml:77

yaml[octal-values]: Forbidden implicit octal value "0600"
tasks/sshconfig.yml:131

yaml[octal-values]: Forbidden implicit octal value "0600"
tasks/sshconfig.yml:208

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/sshconfig.yml:235

yaml[octal-values]: Forbidden implicit octal value "0440"
tasks/sudo.yml:12

yaml[octal-values]: Forbidden implicit octal value "0440"
tasks/sudo.yml:24

yaml[octal-values]: Forbidden implicit octal value "0440"
tasks/sudo.yml:36

yaml[octal-values]: Forbidden implicit octal value "0440"
tasks/sudo.yml:46

yaml[octal-values]: Forbidden implicit octal value "0440"
tasks/sudo.yml:62

yaml[octal-values]: Forbidden implicit octal value "0440"
tasks/sudo.yml:72

yaml[octal-values]: Forbidden implicit octal value "0440"
tasks/sudo.yml:82

yaml[octal-values]: Forbidden implicit octal value "0440"
tasks/sudo.yml:94

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/sudo.yml:121

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/systemdconf.yml:8

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/systemdconf.yml:24

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/timesyncd.yml:8

yaml[octal-values]: Forbidden implicit octal value "0640"
tasks/ufw.yml:66

yaml[octal-values]: Forbidden implicit octal value "0755"
tasks/umask.yml:18

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/umask.yml:43

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/umask.yml:69

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/umask.yml:83

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/umask.yml:94

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/umask.yml:106

yaml[octal-values]: Forbidden implicit octal value "0644"
tasks/umask.yml:120

yaml[octal-values]: Forbidden implicit octal value "0750"
tasks/users.yml:31

Read documentation for instructions on how to ignore specific rule violations.

                   Rule Violation Summary                    
 count tag                profile rule associated tags       
     2 yaml[line-length]  basic   formatting, yaml (warning) 
   111 yaml[octal-values] basic   formatting, yaml           

Failed after min profile: 111 failure(s), 2 warning(s) on 69 files.
```

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console
ansible-lint 6.13.1 using ansible 2.14.2
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

Add a basic `.yamllint` in a Ansible role directory.

**Comment by ssbarnea** [comment]
That is not a bug, by design ansible-lint will respect the yamllint custom config file.
